.prettierignore

```diff
@@ -0,0 +1 @@
+.env-cmdrc
```

Comment:
We needed it because changes in `.env-cmdrc` caused errors in lint-staged.

Reply:
I'd like to fix this. 👍

Reply:
@ota-meshi 

OK, I just looked at `env-cmd`'s document, a [`.env-cmdrc`](https://github.com/toddbluhm/env-cmd#rc-file-usage file) is definitely a JSON file, the problem is our default prettier setting does not recognize `.*rc` files while my `@1stg/lint-staged` config knows that and will try to run `prettier --write` on `.env-cmdrc`.

Fix:

1. write `overrides` in `prettier.config.js`
2. use `@1stg/lint-staged` compatible `@1stg/prettier-config`
3. remove `@1stg/lint-staged` and adopt the verbose config

HDYT?

Reply:
I think `.env-cmdrc` is a JSON file as you say.
I didn't know that, but I think configuring overrides works fine.
https://prettier.io/docs/en/configuration.html#setting-the-parserdocsenoptionshtmlparser-option

I'm not sure what lint-staged configuration needs to be fixed. That's because I'm not familiar with lint-staged 😓 .

Reply:
> I'm not sure what lint-staged configuration needs to be fixed. That's because I'm not familiar with lint-staged 😓 .

`@1stg/lint-staged` is enabled and it recognizes `.env-cmdrc` as JSON file at https://github.com/1stG/configs/blob/master/packages/lint-staged/base.js#L18

> I didn't know that, but I think configuring overrides works fine.

Sure `overrides` will work as expected, see also

https://github.com/1stG/configs/blob/bda0754fa09108d8aef1a526acbf3cd44a5b33af/packages/prettier-config/base.js#L35-L41

---

Conclusion: so we are going to use `overrides` in `prettier.config.js` manually, right? I'd like to raise a PR for it.